src/App.js

```diff
@@ -8,7 +8,13 @@ import SubHeader from "./components/SubHeader";
 import TopNav from "./components/TopNav";
 
 function App(props) {
-    const product = props.state.products[0];
+    // const product = props.state.products[0];
+
+  const filteredProducts = props.state.products.filter((category) => {
```

Comment:
need to go back and study the filter function really well to understand exactly what is happening. 
I wouldn't call the argument to the callback function "category", it is suppose to represent one product from the array per loop